### PR TITLE
[pt-br] Added translated content for regular expression glossary

### PR DESCRIPTION
**Pull request by nathipg**
### Description

Added new page

### Motivation

Help to add pt-br pages to MDN

### Additional details

NA

### Related issues and pull requests

NA

### CODE REVIEW

File: files/pt-br/glossary/regular_expression/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Expressão regular
+slug: Glossary/Regular_expression
+---
+
+{{GlossarySidebar}}
+
+**Expressões regulares** (ou _regex_) são regras que definem quais sequências de caracteres aparece em uma busca.
+
+Expressões regulares são implementadas em várias linguagens, mas a implementação mais conhecida é a do Pearl, a qual deu origem ao seu próprio ecossistema de implementações chamado de PCRE (_Perl Compatible Regular Expression_). Na Web, o {{glossary("JavaScript")}} provê outra implementação da regex por meio do objeto {{jsxref("RegExp")}}.
```
Comment (josielrocha) [inline files/pt-br/glossary/regular_expression/index.md:10]:
```suggestion
Expressões regulares são implementadas em várias linguagens, mas a implementação mais conhecida é a do Perl, a qual deu origem ao seu próprio ecossistema de implementações chamado de PCRE (_Perl Compatible Regular Expression_). Na Web, o {{glossary("JavaScript")}} provê outra implementação da regex por meio do objeto {{jsxref("RegExp")}}.
```

**Review by josielrocha — CHANGES_REQUESTED** [review]
Obrigado pela ajuda @nathipg!
Deixei um pequeno apontamento, poderia verificar por favor?

File: files/pt-br/glossary/regular_expression/index.md (hunk 1 above)
Comment (nathipg) [reply files/pt-br/glossary/regular_expression/index.md]: Boa!

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Pull request merged**